### ISSUE TITLE
Integrity errors after upgrade

### ISSUE THREAD

**Issue by phlax** [translate/pootle#5246]
After upgrading one of our servers we see errors like this when editing units

```
IntegrityError: (1452, 'Cannot add or update a child row: a foreign key constraint fails (`foodb`.`pootle_statistics_scorelog`, CONSTRAINT `submission_id_refs_id_7785fa24` FOREIGN KEY (`submission_id`) REFERENCES `pootle_app_submission` (`id`))')
```

**Comment by phlax** [comment]
i think this is essentially the same issue as #5730

**Comment by phlax** [comment]
closed in favour of #5730
